yarn.lock

```diff
@@ -4492,7 +4492,7 @@ __metadata:
   resolution: "@backstage/eslint-plugin@workspace:packages/eslint-plugin"
   dependencies:
     "@backstage/cli": "workspace:^"
-    "@manypkg/get-packages": "npm:^1.1.3"
+    "@manypkg/get-packages": "npm:^2.2.2"
```

Comment:
Does this fix the original performance issue? I did a quick test locally with `@manypkg/get-packages` and for this repo it's only ~15ms or so to discover all packages.